package-examples/mysql-kustomize/README.md

```diff
@@ -18,8 +18,7 @@ To connect to the database:
 
 To edit the package:
 
-    kpt cfg list-setters mysql-kustomize
-    kpt cfg set mysql-kustomize SETTER VALUE
+    kpt fn eval --image gcr.io/kpt-fn/apply-setters:VERSION mysql-kustomize --  [setter_name=setter_value]
```

Comment:
I imagine people will be copy/pasting these commands. Would it be better to put an actual version here (and add a note that they can change the tag), rather than have the VERSION placeholder?

Reply:
Good feedback. I have updated the readme's based on @phanimarupaka 's template.